cg/meta/store/base.py

```diff
@@ -27,12 +78,77 @@ def add_new_analysis(bundle_data, case_obj, status, version_obj):
     return new_analysis
 
 
+def parse_config(data: dict) -> dict:
```

Comment:
```suggestion
def parse_mip_config(data: dict) -> dict:
```
Same reason here if we want to go in this direction

Reply:
Also resolved in https://github.com/Clinical-Genomics/cg/pull/621/commits/a6295bbeb481265db210bcf6eb8fbe9deaae79dc